hawkauthlib/__init__.py

```diff
@@ -95,12 +107,17 @@ def get_id(request, params=None):
 
 
 @utils.normalize_request_object
-def get_signature(request, key, algorithm=None, params=None, accept_untrusted_content=False):
+def get_signature(request, key, include_payload_hash=True, algorithm=None, params=None):
```

Comment:
The name `accept_untrusted_content` didn't seem quite right here, because when you're calculating a signature you're not *necessarily* making any trust decisions (e.g. if you're a client). I've renamed it and moved it up to a more prominent position in the argument list, since it "overrides" values in `params`.

Reply:
you're assuming the trust decisions are not in the hawkauthlib already, in fact they are! You inherently trust client provided hash, without verifying it currently. i.e `accept_untrusted_content` is only needed _because_ historical API of hawkauthlib included a trust decision and the new API implements the expected API that gains trust through verification. So `accept_untrusted_content` allows users to continue with usage as they did before this patch.